docs/what-is-bitcoin.md

```diff
@@ -0,0 +1,71 @@
+## Overview
+
+Bitcoin is a peer-to-peer digital currency and network. Bitcoins can be sent and received through
+the internet, and aren't centrally controlled by anyone. Unlike using a bank, this means nobody
+can seize, block or take control of your money. Instead, Bitcoin is run and verified collectively by
+its users, as a peer-to-peer network. Let's get started and explain the history behind Bitcoin
+and how it works.
+
+## Background and transactions
+
+Bitcoin was first described in the Bitcoin whitepaper, by Satoshi Nakamoto, its first developer.
+Before Bitcoin, digital currencies and payment systems all relied on trust, by either trusting a
+company, or an individual. This made it easy for these trusted persons to block people's money,
+or succumb to Government intervention. Satoshi figured out a way to overcome that, by inventing
```

Comment:
LINE: 14 Change "Satoshi figured out a way to overcome that," to "Satoshi engineered a way to overcome these unethical economic policies by inventing" - I think we can all agree that a lot of national economic policies are unethical and undermine the interests of The People.